plugin/repeater/repeater.talon

```diff
@@ -3,3 +3,6 @@
 <number_small> times: core.repeat_command(number_small - 1)
```

Comment:
Should we have the phrase form of this one?
`phrase  <number_small> times`

Reply:
That would conflict with

https://github.com/knausj85/knausj_talon/blob/main/core/text/text.talon#L2-L7